### PR TITLE
delete all todo items

### LINKED ISSUE #6: Support to delete all todo items in one go
We need to provide an endpoint to delete all todo items in one go.

### PR DESCRIPTION
**Pull request by ghandmann**
Fixes #6

### CODE REVIEW

**Review by sutterseba — APPROVED** [review]

File: app.js [hunk 1]
```diff
@@ -42,6 +42,11 @@ app.delete("/todo-items/:todoItemId", (req, res) => {
     res.status(200).send();
 });
 
+app.delete("/all", (req, res) => {
```
Comment (ghandmann) [inline app.js:45]: Why not use `/delete-all-todo-items` as endpoint?

File: app.js [hunk 2]
```diff
@@ -42,6 +42,11 @@ app.delete("/todo-items/:todoItemId", (req, res) => {
...
+    this.inMemoryStore = [];
+    return res.status(200).send();
+});
+
 websocketServer.on('connection', (clientWebsocket) => {
     console.log("new websocket client connected.");
```
Comment (ghandmann) [inline app.js:52]: And where are the tests???

**Review by ghandmann — COMMENTED** [review]
There are open issues.

File: app.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ app.inMemoryStore = [];
 
 // Gibt Liste aller TodoItem zurück
 app.get("/todo-items/", (req, res) => {
-    res.send(app.inMemoryStore);
+    // res.send(app.inMemoryStore);
```
Comment (sutterseba) [inline app.js]: This doesnt seem right

**Review by sutterseba — CHANGES_REQUESTED** [review]

File: app.js (hunk 3 above)
Comment (ghandmann) [reply app.js]: Good catch! Going to fix...

File: app.js (hunk 1 above)
Comment (ghandmann) [reply app.js:45]: don't like this.

File: app.js (hunk 2 above)
Comment (ghandmann) [reply app.js:52]: tests have been added.

**Pull request merged**